pkg/operator/certrotationcontroller/certrotationcontroller.go

```diff
@@ -322,6 +322,47 @@ func NewCertRotationController(
 	}
 	ret.certRotators = append(ret.certRotators, certRotator)
 
+	certRotator, err = certrotation.NewCertRotationController(
+		"LocalhostRecoveryServing",
+		certrotation.SigningRotation{
+			Namespace:     operatorclient.OperatorNamespace,
+			Name:          "localhost-recovery-serving-signer",
+			Validity:      10 * 365 * defaultRotationDay, // this comes from the installer
+			Refresh:       8 * 365 * defaultRotationDay,  // this means we effectively do not rotate
+			Informer:      kubeInformersForNamespaces.InformersFor(operatorclient.OperatorNamespace).Core().V1().Secrets(),
+			Lister:        kubeInformersForNamespaces.InformersFor(operatorclient.OperatorNamespace).Core().V1().Secrets().Lister(),
+			Client:        kubeClient.CoreV1(),
+			EventRecorder: eventRecorder,
+		},
+		certrotation.CABundleRotation{
+			Namespace:     operatorclient.OperatorNamespace,
+			Name:          "localhost-recovery-serving-ca",
+			Informer:      kubeInformersForNamespaces.InformersFor(operatorclient.OperatorNamespace).Core().V1().ConfigMaps(),
+			Lister:        kubeInformersForNamespaces.InformersFor(operatorclient.OperatorNamespace).Core().V1().ConfigMaps().Lister(),
+			Client:        kubeClient.CoreV1(),
+			EventRecorder: eventRecorder,
+		},
+		certrotation.TargetRotation{
+			Namespace: operatorclient.TargetNamespace,
+			Name:      "localhost-recovery-serving-certkey",
+			Validity:  10 * 365 * rotationDay,
```

Comment:
I think both should be `defaultRotationDay` we don't want to tweak this value ever.

Reply:
fixed